packs/orion/actions/ncm_config_download.py

```diff
@@ -27,6 +27,10 @@ def run(self, node, platform, configs):
         results = {}
         node_ids = []
 
+        # Split into an Array, as enum and array type does not play nice
+        # in the WebUI.
+        configs = configs.split(",")
```

Comment:
Other changes look good so I will go ahead and merge it into master, but I would appreciate if you can open up a new PR when this parameter type is changing.

The problem is that not using the native type in the metadata kinda breaks the whole abstraction so now user needs to know implementation detail (either check the code or description aka something which can't be done automatically) to know what type the parameter is.